examples/06-verif-manual/vm-009-large_lateral_deflection_of_unequal_stiffness_springs.py

```diff
@@ -0,0 +1,409 @@
+# Mass location:
+# Mass at node J
+mapdl.keyopt(3, 6, 2)
+
+# Element type COMBIN40.
+mapdl.et(4, "COMBIN40")
+
+# Special Features are defined by keyoptions of the element COMBIN40.
+# KEYOPT(3)(2)
+# Element degrees of freedom:
+# UX (Displacement along nodal X axes)
+mapdl.keyopt(4, 3, 2)
+
+# KEYOPT(6)(2)
+# Mass location:
+# Mass at node J
+mapdl.keyopt(4, 6, 2)
+
+# Print the list of the elements and their attributes.
+print(mapdl.etlist())
+
+# Print the list of the elements and their attributes.
+print(mapdl.elist())
+
+
+###############################################################################
+# Define Real Constants
+# ~~~~~~~~~~~~~~~~~~~~~
+# Define damping coefficients :math:`c_x = 1.41`, :math:`c_y = 2.0` and
+# stiffness values :math:`k_1 = 8\,N/cm`, :math:`k_2 = 1\,N/cm` for the
+# spring elements.
+
+# Define real constant 1 with stiffness k2.
+mapdl.r(nset=1, r1=k_spring2)  # SPRING STIFFNESS = 1
+
+# Define real constant 2 with stiffness k1.
+mapdl.r(nset=2, r1=k_spring1)  # SPRING STIFFNESS = 8
+
+# Define real constant 3 with damping coef. in X-direction and mass.
+mapdl.r(nset=3, r2=c_damp_x, r3=mass)
+
+# Define real constant 4 with damping coef. in y-direction and mass.
+mapdl.r(nset=4, r2=c_damp_y, r3=mass)
+
+# Print the real constant list.
+print(mapdl.rlist())
+
+
+###############################################################################
+# Define Nodes
+# ~~~~~~~~~~~~
+# Set up the nodes coordinates using python ``for-loop``.
+
+# Lists with nodes coordinates.
+node_x_coord = [0, 0, 0, -1, 0]
+node_y_coord = [0, 10, 20, 10, 9]
+
+# Create nodes.
+for i in range(0, 5):
+    mapdl.n(node=i+1, x=node_x_coord[i], y=node_y_coord[i])
+
+# Print the list of the created nodes.
+print(mapdl.nlist())
+
+
+###############################################################################
+# Create Elements
+# ~~~~~~~~~~~~~~~
+# Create the elements through the nodes.
+
+# Create  spring element COMBIN14 between nodes 1 nad 2
+# with stiffness k_2 = 1 N/cm.
+mapdl.type(1)
+mapdl.real(1)
+mapdl.e(1, 2)
+
+# Create  spring element COMBIN14 between nodes 2 nad 3
+# with stiffness k_1 = 8 N/cm.
+mapdl.type(1)
+mapdl.real(2)
+mapdl.e(2, 3)
+
+# Create  spring element COMBIN40 between nodes 4 nad 2
+# with damping coefficient c_x = 1.41.
+mapdl.type(3)
+mapdl.real(3)
+mapdl.e(4, 2)
+
+# Create  spring element COMBIN40 between nodes 5 nad 2
+# with damping coefficient c_y = 2.0.
+mapdl.type(4)
+mapdl.real(4)
+mapdl.e(5, 2)
+
+# Print the list of the created elements.
+print(mapdl.elist())
+
+
+###############################################################################
+# Define Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application of boundary conditions (BC) for the spring model.
+
+# Unselect the node where the force is applied.
+mapdl.nsel("U", "NODE", vmin=2)
+
+# Apply BC to the selected set of the nodes.
+mapdl.d("ALL", "ALL")
+mapdl.nsel("ALL")
+
+# Finish pre-processing mode.
+_ = mapdl.finish()
```

Comment:
```suggestion
mapdl.finish(mute=True)
```

Please use ``mute``.